tests/sample_data/multimodel_statistics/test_multimodel.py

```diff
@@ -24,7 +25,7 @@
         marks=pytest.mark.skip(
             reason='Cannot calculate statistics with single cube in list')),
     '365_day',
-    'gregorian',
+    'standard' if cf_units.__version__ >= '3.1' else 'gregorian',
```

Comment:
this is a spanner in the works - we shouldn't test like this, we should mark them as xfailed if the version is lower than 3.1 since ideally that'll not happen

Reply:
Note that the other tests are also written to support both the new and older versions of cf-units

Reply:
the tests are, but passing is not the point of a test, is the functionality in our code dual as well? If yes, I is happy and no more nagging :grin:

Reply:
It looks like `'gregorian'` as a calendar was removed in v1.9 of the CF-conventions. I'll update the code too so it does something sensible..

Reply:
I think the code works as is with both versions, it's just that versions of cf-units >= 3.1 will automatically convert the `'gregorian'` calendar to `'standard'`.

Reply:
OK sounds good, thanks, bud :beer: